sdk/storage/azure-storage-file/azure/storage/file/_generated/operations/_share_operations.py

```diff
@@ -293,6 +293,123 @@ def create_snapshot(self, timeout=None, metadata=None, cls=None, **kwargs):
             return cls(response, None, response_headers)
     create_snapshot.metadata = {'url': '/{shareName}'}
 
+    def create_permission(self, timeout=None, cls=None, **kwargs):
```

Comment:
>def create_permission(self, timeout=None, cls=None, **kwargs): [](start = 4, length = 62)

It seems the swagger is not correct? the permission should be in request body, currently it looks like the quest body is empty?

Reply:
Correct - your current swagger files don't have the JSON permission body.

Reply:
@tg-msft thanks for the clarification! We can still pass in the serialized body(done through hand-written code?), right?

Reply:
I'm not sure -- @annatisch could answer that better.

My team is actively investigating if we can have it generate the JSON serialization for you _(with a future swagger update, of course)_.  I'd suggest you hold off on the CreatePermission/GetPermission APIs like .NET is doing until we can loop back around with you on that.

Reply:
Sounds good. Let's merge as is, and fix up the REST parity work later